src/main/kotlin/frc/robot/subsystems/DrivetrainSubsystem.kt

```diff
@@ -9,8 +9,10 @@ import com.kauailabs.navx.frc.AHRS
  * TODO: add gyro and encoder handling
  */
 class DrivetrainSubsystem(val drivetrain: DifferentialDrive, val gyroscope: AHRS) : SubsystemBase() {
+    // Accepts the drivetrain type (tank), and the gyroscope type.
```

Comment:
Again, put comments above code the describe.

Also, this comment is unnecessary. The code itself clearly describes its arguments and their type.